apps/resume-portal/.gitignore

```
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

tsk tsk, this is wrong, should be
```suggestion
node_modules/
```